AdblockPlusSafari/AdblockPlusController.m

```diff
@@ -100,6 +115,20 @@ -(UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NS
   return cell;
 }
 
+- (NSString *)tableView:(UITableView *)tableView titleForFooterInSection:(NSInteger)section
+{
+  NSDate *lastUpdate = self.adblockPlus.lastUpdate;
+  if (section == 0 && lastUpdate != nil) {
+    NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+    dateFormatter.locale = [NSLocale currentLocale];
+    dateFormatter.dateStyle = NSDateFormatterFullStyle;
+    dateFormatter.timeStyle = NSDateFormatterShortStyle;
+    return [NSString stringWithFormat:@"Last filterlist update: %@", [dateFormatter stringFromDate:lastUpdate]];
```

Comment:
Same as with the variable name, "filter list" should be two words - that was wrong in the mockup.